Sources/GDSCommon/Utilities/GDSLocalisedString.swift

```diff
@@ -84,3 +84,11 @@ extension GDSLocalisedString: CustomStringConvertible {
         value
     }
 }
+
+extension GDSLocalisedString: Equatable {
+    public static func == (lhs: GDSLocalisedString, rhs: GDSLocalisedString) -> Bool {
+        lhs.stringKey == rhs.stringKey &&
+        lhs.variableKeys == rhs.variableKeys &&
+        lhs.bundle == rhs.bundle
```

Comment:
I think we should check for `attributes` too? Indeed, the `let attributes: Attributes?` should be further up the file with the other properties to avoid it being missed.

Reply:
Looking into it now, I think it would also be complex to add and might need to be a breaking change 😬 
Attributes is using a tuple currently, which can't conform to Equatable, so it's not trivial to compare.

Reply:
You could make do a check on whether attributes is `nil` or not. Not perfect, but better than nothing?

This is a bit ugly and could probably be improved but I think would work:

```
((lhs.attributes == nil ) == (rhs.attributes == nil ) ||
         (lhs.attributes != nil ) == (rhs.attributes != nil ))
```